### DISCUSSION TITLE
error in running external node

### DISCUSSION THREAD

**Discussion by cliff0412** [zkSync-Community-Hub/zksync-developers#45] [General]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement.

i am trying to run an external node as documented here
https://github.com/matter-labs/zksync-era/blob/main/docs/external-node/01_intro.md

### What is the specific issue or error you're encountering?

```
error: The following required arguments were not provided:
    --database-url <DATABASE_URL>
```

my command is
```docker run matterlabs/external-node-public:v0.0.4 --env-file /root/zksync_nodes/mainnet-config.env --mount type=bind,source=/data/rocksdb/,target=./db/ext-node/ --env DATABASE_URL=postgres://postgres@localhost/zksync_local_ext_node
```

### Can you share the error messages or logs you're receiving, if any?

error: The following required arguments were not provided:
    --database-url <DATABASE_URL>

### Have you made any recent changes to the contract before encountering this issue?

no

### Are there any external libraries or contracts that your contract interacts with?

no

### Can you provide the relevant portions of your contract code where the issue is occurring?

no

### Have you tried to isolate the problem, and if so, what were the results?

no

### What steps have you already taken to try to resolve the issue?

no

### Repo Link (Optional)

https://github.com/matter-labs/zksync-era/issues/50

**Comment by uF4No** [comment]
I'm closing this thread as you've already created an issue in the zksync-era repo.
